lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -610,7 +610,7 @@ L<Mojolicious::Controller/"send">.
     <head><title>Echo</title></head>
     <body>
       <script>
-        var ws = new WebSocket('<%= url_for('echo')->to_abs %>');
+        var ws = new WebSocket('<%= url_for('echo')->to_abs->scheme('ws') %>');
```

Comment:
I don't see why this is required. The websocket route defines the scheme as well.